packages/docusaurus-theme-common/src/components/Collapsible/index.tsx

```diff
@@ -119,7 +118,7 @@ function useCollapseAnimation({
 
     // On mount, we just apply styles, no animated transition
     if (!mounted.current) {
-      applyCollapsedStyle(el, collapsed);
+      queueMicrotask(() => applyCollapsedStyle(el, collapsed));
```

Comment:
```suggestion
      applyCollapsedStyle(el, collapsed);
```

local test commited by mistake